pkg/systemd/index.html

```diff
@@ -156,7 +156,7 @@
                     </ul>
                 </div>
 
-                <label class="control-label" translatable="yes">Performance Profile</label>
+                <label class="control-label" for="system-info-performance" translatable="yes">Performance Profile</label>
                 <span id="system-info-performance" hidden></span>
```

Comment:
There is no labelable element in pkg/tuned/link.html, it's just a link. So I think in this case we should rather drop the  control-label class?